webnn/resources/test_data/hard_sigmoid.json

```diff
@@ -0,0 +1,542 @@
+{ // hardSigmoid: Calculate the non-smooth function used in place of a sigmoid function on the input tensor.
+  "tests": [
+    {
+      "name": "hardSigmoid float32 1D tensor default options", // default options: {alpha: 0.2, beta: 0.5}
+      "inputs": {
+        "x": {
+          "shape": [24],
+          "data": [
+            0.962285784541856,
+            0.2570918415891237,
+            0.5411237762597576,
+            -0.7982109817680292,
```

Comment:
Beware that mixing negative and positive numbers in addition can cause subtraction of nearly equal numbers, meaning catastrophic cancellation and potentially exceeding the ULP. Consider using positive inputs here and other cases. Similarly using a negative scale with positive beta could yield the same issue (unless the inputs were negative, in which case the inputs and scale would cancel signs, and so that would be fine).

clamp(a * alpha + beta, 0, 1));   defaults alpha = 0.2, beta = 0.5

Reply:
Done to update test data to avoid catastrophic cancellation, please take another look, thanks! @fdwr